p1

```diff
@@ -0,0 +1,2 @@
+Identity added: /home/mikuhnt/.ssh/id_rsa (/home/mikuhnt/.ssh/id_rsa)
```

Comment:
this is probably not meant to be here :smile: 

There is also a ```garm``` and ```garm-cli``` binary commited.

Reply:
Sorry.... I rebased mistakenly on another branch. Fixed.